Makefile

```diff
@@ -123,4 +132,36 @@ cluster/prepare/local: cluster/prepare
 	-oc create -f deploy/crds/*.crd.yaml
 	@oc create -f deploy/service_account.yaml
 	@oc create -f deploy/role.yaml
-	@oc create -f deploy/role_binding.yaml
\ No newline at end of file
+	@oc create -f deploy/role_binding.yaml
+
+.PHONY: cluster/prepare/olm
+cluster/prepare/olm: cluster/prepare
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-source-template.yml | oc create -f - -n openshift-marketplace
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-subscription-template.yml | oc create -f - -n $(NAMESPACE)
+	timeout 1m bash -c 'until oc get crd installations.integreatly.org; do sleep 5; done'
```

Comment:
@philbrookes Added this line to check the crd exists before continuing. Unsure if this will work on those Mac things, can you give it a go?